### PR TITLE
First pull request for Android version.

### PR DESCRIPTION
**Pull request by Jiki-Mo**
This pull request only modifies the software icon and software properties, and no changes are made to the UI.

### CODE REVIEW

**Review by rotorflight — CHANGES_REQUESTED** [review]
You MUST NOT change the yarn.lock. Especially not with npm, as it is for yarn!

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "rotorflight-configurator",
   "productName": "Rotorflight Configurator",
   "description": "Crossplatform configuration tool for Rotorflight flight control system.",
-  "version": "0.0.0",
```
Comment (rotorflight) [inline package.json:5]: Please don't change this. It is automatically set to the right version in the build process. Otherwise it should be 0.0.0.

File: package.json [hunk 2]
```diff
@@ -65,7 +65,6 @@
     "lru_map": "^0.3.3",
     "marked": "^4.0.17",
     "multiple-select": "^1.5.2",
-    "nw-vue-devtools-prebuilt": "^0.0.10",
```
Comment (rotorflight) [inline package.json:68]: Why is this removed?

**Comment by Jiki-Mo** [comment]
> 您不得更改 yarn.lock。尤其不能使用 npm，因为它是为 yarn 设计的！



> You MUST NOT change the yarn.lock. Especially not with npm, as it is for yarn!

No, this file is automatically generated when the software is installed and is not modified manually.

File: package.json (hunk 1 above)
Comment (Jiki-Mo) [reply package.json:5]: OK, I will restore this value to the default.

File: package.json (hunk 2 above)
Comment (Jiki-Mo) [reply package.json:68]: I cannot install the software nw-vue-devtools-prebuilt. The function of this software is to debug applications. I will try again. If it still cannot be installed, I will give up on it because I want to see the result of the application. I will directly generate a debugging version of osx64, which only takes 15 seconds.

**Comment by rotorflight** [comment]
Merged the Adndroid files manually. Common files not merged, as they would break the build for other platforms.
